### DISCUSSION TITLE
<Your question or feedback about the GitHub Discussions product>

### DISCUSSION THREAD

**Discussion by LocaB-43** [community/community#65717] [Discussions]
### Select Topic Area

Question

### Body

How do I trace my hackers

**Comment by AmberSolstice** [comment]
Look at your firewall activity

**Comment by Akash1134** [comment]
Hi @LocaB-43 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
